### ISSUE TITLE
oc status should let me know if all the pods for a DC are pending

### ISSUE THREAD

**Issue by deads2k** [openshift/origin#6426]
It's annoying that I can't look at status and know if my service or pod should be accessible.  If that pod isn't running, its not very useful to me.

```
oc status
In project mythtv-backend on server https://eads-nas:8443

dc/tmp deploys 172.30.201.81:5000/mythtv-backend/mythtv-backend-base:latest 
  #1 deployment running for 10 seconds - 1 pod

View details with 'oc describe <resource>/<name>' or list everything with 'oc get all'.
```

**Comment by kargakis** [comment]
+1

I would expect "deployment running" to show up when at least one pod is ready. Please assign to me so I won't forget about it.

**Comment by kargakis** [comment]
@deads2k "deployment running" here means the deployer. Shoule we reword it to something like "deployment process running"?

**Comment by deads2k** [comment]
I'm ok with "deployment running".  I'm frustrated that having "1 pod" can be in any state.  I think that only `Running` should be considered "1 pod"  Maybe just listing the state is sufficient.

**Comment by kargakis** [comment]
After thinking about this more, I think the best solution would be to surface running pods in the DC status (similar to https://github.com/kubernetes/kubernetes/pull/19382) and re-use that. See https://github.com/openshift/origin/pull/6443#issuecomment-173267479

**Comment by kargakis** [comment]
> After thinking about this more, I think the best solution would be to surface running pods in the DC status (similar to kubernetes/kubernetes#19382) and re-use that. See #6443 (comment)

Opened https://github.com/openshift/origin/issues/6743

**Comment by kargakis** [comment]
> Blocked on #5918

Finally fixed

**Comment by kargakis** [comment]
We should use [rc.spec.readyReplicas](https://github.com/kubernetes/kubernetes/pull/29481) for this - coming with the next rebase.
